app/client/components/SpeedIndicator.js

```diff
@@ -7,6 +7,12 @@ export default class SpeedIndicator extends React.Component {
         speed: React.PropTypes.number.isRequired,
     }
 
+    speedClicked() {
```

Comment:
I suggest to move this method after `render()`.
it is just a code organisation thing but I believe a component looks nice when the methods are structured like this:

- propTypes
- defaultProps
- state
- lifecycle methos (componentWillMount, ...)
- render() 
- callbacks / APIs methods